docs/design/plugin_structure.md

```diff
@@ -27,20 +26,21 @@ Initialize processing is like the following.
 4. Initialize the plugin according to the merged config, which is created by the input config and the default config.
 
 
+
 ## Plugin usage in Satellite
+Nowadays, we have 9 kinds of plugins in Satellite. 2 kinds of Plugins are the [sharing plugins](module_design.md) and the others are the [normal plugins](module_design.md).
+
+
 
 - Extension: 
-    - Collector interface 
-        - the specific plugin: segment-receiver
-    - Queue interface
-        - the specific plugin: mmap-queue
-    - Filter interface
-        - the specific plugin: sampling-filter
-    - Client interface
-        - the specific plugin: gRpc-client
-    - Fallbacker interface
-        - the specific plugin: timer-fallbacker
-    - Forwarder interface
-        - the specific plugin: grpc-segment-forwarder
-    - Parser interface
-        - the specific plugin: csv-parser
+    - sharing plugins
+        - Server interface
+        - Client interface
```

Comment:
Let's don't use `interface`, this is not a design concept, this is the code. You called them as plugins in other parts of the doc.